eclipse-collections/src/main/java/org/eclipse/collections/impl/bimap/mutable/SynchronizedMutableBiMap.java

```diff
@@ -389,7 +393,7 @@ public ShortIterable collectShort(ShortFunction<? super V> shortFunction)
     }
 
     /**
-     * @deprecated in 7.1. Use {@link OrderedIterable#zip(Iterable)} instead.
+     * @deprecated in 8.0. Use {@link OrderedIterable#zip(Iterable)} instead.
```

Comment:
I coded this back when 7.1 wasn;t released, and I committed it that way. Later I changed everything to be 8.0. SynchronizedBiMap didn't exist in 7.1, and neither did the deprecation of these methods in the BiMap heirarchy.